lib/assets/data_table/main.js

```diff
@@ -118,6 +125,13 @@ export function init(ctx, data) {
         const [key, order] = reorder(this.data.content.order_by, this.data.content.order, newKey)
         ctx.pushEvent("order_by", { key, order });
       },
+
+      orderDescription() {
+        const key = this.data.content.order_by
+        const label = this.data.content.columns[key].label
```

Comment:
I can change it if you prefer. But I'm basically doing both at once. Fetching the column from the array and accessing the corresponding label.
Now that I removed the conditional, this breaks if the key is not in the array, but this code is only called when we know the key is there.

Reply:
Or maybe I didn't pay enough attention and not always the key of the column will be its position… 🤔
My turn to be confused!

Reply:
On the elixir side the keys can be of arbitrary data type, so we map them to strings when sending to the client. It happens that we generate those strings sequentially, so they make for valid indices in most cases. However, in cases like ETS the columns may change as we go through pages, and then we generate more keys and they are no longer valid indices. Either way, we shouldn't rely on what those keys are, so instead let's do `columns.find((column) => column.key === key)` :)

Reply:
Note there is an option issue #74 to make the tables less strict. In particular, should be printed as tables of a single column where the column is the value itself. This should simplify a lot of things internally. :) But for now we need to handle it, yeah. :)